src/components/TrackListItem.demo.js

```diff
@@ -0,0 +1,32 @@
+import Demo, {props as P} from 'react-demo'
+// See https://github.com/rpominov/react-demo for how to make a demo
+
+import React from "react";
+
+import TrackListItem from "./TrackListItem";
+
+export default (<Demo
+  props={{
+    trackType: P.choices(["graph", "haplotype", "read"]),
+    availableTracks: P.json([{"files": [{"name": "fileA1.vg", "type": "graph"},
+                                        {"name": "fileA2.gbwt", "type": "haplotype"}]},
+                             {"files": [{"name": "fileB1.gbwt", "type": "haplotype"},
+                                        {"name": "fileB2.gam", "type": "read"}]},
+                             {"files": [{"name": "fileC1.xg", "type": "graph"}]}]),
```

Comment:
Shouldn't we start with a `trackFile` and `trackColorSettings` here as well?

Reply:
I struggled a bit with the design of default variables, and whether/when to use them. I wasn't sure which components should have them and what props should have a default. I omitted them here out of readability I suppose.